packages/libluajit/build.sh

```diff
@@ -3,8 +3,9 @@ TERMUX_PKG_DESCRIPTION="Just-In-Time Compiler for Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1:2.1.0-beta3
-TERMUX_PKG_SRCURL=https://github.com/LuaJIT/LuaJIT/archive/v${TERMUX_PKG_VERSION:2}.tar.gz
-TERMUX_PKG_SHA256=409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/LuaJIT/LuaJIT/archive/787736990ac3b7d5ceaba2697c7d0f58f77bb782.tar.gz
```

Comment:
Is there a better way to do this?

Does `TERMUX_PKG_VERSION` need change ?
Also what's with the first 1: in 1:2.1.0-beta3.

The 2.1.0-beta3 tag is from 2017 . And neovim depends on newer version.
Neovim's build system currently uses that commit to build .

One of neovim dev told me segfaults
are expected if neovim is built with older version of luajit . So I updated it . Though it doesn't completely solve the segfault .

Reply:
This works, you could add the commit hash as `_COMMIT=787736990ac3b7d5ceaba2697c7d0f58f77bb782`, but it is not really necessary. 

> Also what's with the first 1: in 1:2.1.0-beta3.

That's an [epoch](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version), needed when changing between version formats so that apt knows to consider the new format as newer compared to the old one

Reply:
This is a good idea. I'll add a `TERMUX_PKG_COMMIT`